crates/ra_syntax/test_data/parser/inline/ok/0004_value_parameters_no_patterns.rs

```diff
@@ -1 +1 @@
-type F = Box<Fn(a: i32, &b: &i32, &mut c: &i32, ())>;
```

Comment:
Wait, this was never allowed in the first place? :sweat_smile:

Reply:
Maybe add some code to test all code which should compile with rustc too?

Reply:
That could be a good catch-all, but I think a lot of existing tests won't compile due to undefined symbols, rather than syntax errors.